auto_vk_toolkit/include/configure_and_compose.hpp

```diff
@@ -186,6 +186,7 @@ namespace avk
 			.setShaderUniformTexelBufferArrayDynamicIndexing(VK_TRUE)
 			.setShaderStorageTexelBufferArrayDynamicIndexing(VK_TRUE)
 			.setDescriptorIndexing(VK_TRUE)
+			.setTimelineSemaphore(VK_TRUE)
```

Comment:
This should probably not be enabled by default but instead, enabled through the `std::function<void(vk::PhysicalDeviceVulkan12Features&)>` parameter that can be passed to `configure_and_compose`.

Reply:
✅